README.md

```diff
@@ -1,8 +1,6 @@
 # Getting Started with the .NET MAUI Rating Control
 
-A quick-start project that helps you to integrate a MAUI Rating control in the .NET MAUI application.This project contains code to change the control’s precision mode, modify the rating item shape, restrict interaction in the control, and customize its appearance.
-
-Documentation: https://help.syncfusion.com/maui/rating/getting-started
+The [.NET MAUI Rating](https://www.syncfusion.com/maui-controls/maui-rating?utm_source=github&utm_medium=listing&utm_campaign=maui-rating-github-samples) control allows users to select a rating value from a group of visual symbols like stars, hearts, or custom shapes.
```

Comment:
Recheck on this(existing content shouldn't delete), use the below here
The [.NET MAUI Rating](https://www.syncfusion.com/maui-controls/maui-rating?utm_source=github&utm_medium=listing&utm_campaign=maui-rating-github-samples) control allows users to select a rating value from a group of visual symbols like stars, hearts, or custom shapes. A quick-start project that helps you to integrate a MAUI Rating control in the .NET MAUI application.This project contains code to change the control’s precision mode, modify the rating item shape, restrict interaction in the control, and customize its appearance.